diff --git a/oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java b/oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java
index adcf323..15c3ac6 100644
--- a/oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java
+++ b/oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java
@@ -1102,7 +1102,7 @@ private void setLongSetting(String propName, long val) {
         public final long toMs;
 
         public TimeInterval(long fromMs, long toMs) {
-            checkArgument(fromMs < toMs);
+            checkArgument(fromMs <= toMs);
             this.fromMs = fromMs;
             this.toMs = toMs;
         }
